docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
iB7oUyjrGERERvPXWW7z33nscOHCAxo0bVznKZ/78+YSFhXHo0CGef/55Zs2axT///GPe799//91CsNmwYQPZ2dmMGDGiQv2vWLECjUbD3r17WbJkCYsWLeKrr74yL580aRL79+/nt99+Y/fu3ciyzD333INOp0OSJHr37m3+3aWkpBAVFUVOTg6nTp0ClN9dp06dzJF7o0ePJiEhgb///psDBw7Qvn17+vXrZz4XAM6cOcNPP/3Ezz//XCFPsaysLObMmcP+/fvZtGkTKpWKESNGYDQqInhmZiZDhgyhVatWHDhwgHnz5jF37lzz+iqVinHjxhUrnhgREUGPHj3w9/c3t/vwww85ceIEK1asYPPmzTz77LMW62RnZ7NgwQJWrlzJtm3buHjxosW2li5dyhNPPMG0adM4duwYv/32m8V5Xd7x+eGHH5g3bx5vv/02+/fvx8fHh08//bTYMencuTOXL1++5X6G9VIsQ5JwKvjOyM+/XrNjEVSZopUwATJkO5p62pe1SrXQp6GSQvlLyjF0eRUXEo5cU9LzQm0aFFvm7tocAIMkoTXEozcYi7UR3LmYb7bzsyE/A6wcSLBRTMutNCqcbG5/aLtd/3k8mKmM66sDH7BmxxtclYw00BsY13dBsfYzes5DJctsNqZz/ORac2SZEMvqHqbIsgYF4hb2ilgmyzJRBdEMrSqR0m6lUdHWz50GOqWK6/lrimC87rDygGeU4fXfTyLLMievphMdn4mVWsU9oaWnFJnSMEWBibpNdXmWmdIwo5Ojefjvh3lp50tEXopkbfTamx2ioI6TY7jzDP6Ts/KJT1f+NoIaOBZJw6ycWKbW2qEuEMkqE9F1p3DhwgV8fUve57feeos+ffrQqlUrnn/+eXbt2mWOPlmwYAGTJk1i+vTpBAUFMWfOHEaOHGkR2VWU9PR00tLSGDJkCM2aNSM4OJiJEyfSuHFjcxuj0cjy5csJCQmhV69eTJgwgU2bNgGKkLF06VLmz5/P4MGDadWqFV9++SW2trZ8/fXXgBLhtGPHDgwGA0ePHsXKyorx48ebRZjIyEj69OlT6rG4ePEi/fv3p2XLljRv3pzRo0cTFqZ4zi5atIh+/frx8ssvExQUxKRJk5gxYwbz58+v3AGvxDH+6KOPmDJlCpMnTyYoKIhXXnmF0NDQKm2nR48ePP/88wQFBTFz5kzuv/9+PvjgAwAGDhyIvb09v/xSmM2zatUq7rvvPhwdKxYB7+fnxwcffECLFi0YP348M2fONPcfExPDb7/9xldffUWvXr0ICwsjIiKCK1eumAsghIeHm39P27Zto127dhafFf3d7dixg71797J27Vo6duxI8+bNWbBgAS4uLhbRafn5+Xz77be0a9eONm3alLsPo0aNYuTIkQQGBtK2bVu++eYbjh07xsmTJ83HxGg08vXXX9O6dWuGDBnCM888Y9HH+PHj2blzJxcvXgSUc3rNmjWMHz/e3Gb27NncddddBAQE0LdvX958801++OEHi350Oh2fffYZHTt2pH379syYMcP8twBKxOHTTz/NrFmzCAoKolOnTsyePbvCx2fx4sVMmTKFKVOm0KJFC958801atWpV7JiYrg2mqLhbRb1NTnaUFZ3QqEuq4ZEIqopJLHMsEJbytQ7FjKJvBYO6v8CSVX8Qr5b4bdsrjLp7YbnrGA16NqbFgBo6+XQttlxrrUQ7pqskXDQJZObpcbG7uYcHwe3DnMaRWSC+N+xAQpbiPeflaF1h76dqxbkRDwaPZ/nlXzidfZVzsb+BBNPd2mPj3KhY86ZN+jFolwd/6ZP468R3PDxgKAAJGbkYjXKxyq6C2otZLMsvEKIK0jATMvJIzspHJRU8oFWCzk3cOBplB7bZXEg6zbnrmRy5lIpaJaFWSew6m8TGk/Hsj1VmDvsFe+Fsqy21v4bmyLLsyu6eoBZh8s6rLoP/hJwEEooUmYhJiSl33di0WFysXXCxcbmpMQhqJzm6O08sM0WV+bnZ4ijlKpNwUGmDf1QqtGaRrHAS1lZjy54H91TDSCtPZSL4cnJySk1RKyowmFLiEhISaNy4MVFRUcXM7Hv06MGSJUtK7MvNzY1JkyYxcOBA7r77bvr378+YMWMsUu0CAgIsxBkfHx8SEpRrzdmzZ9HpdPTo0cO8XKvV0rlzZ6KilOrovXr1IiMjg0OHDrFr1y769OlDeHi4OTpt69atxcSNosyZM4dHH32UlStX0r9/f0aPHk2zZs0AiIqKYtiwYcX2d/HixRgMBtRqdan9lkVZx/j06dNMnz7don3nzp3ZvHlzpbfTrVu3Yu9NKaMajYYxY8YQERHBhAkTyMrK4tdff61U+l3Xrl0t7sO7devGwoULMRgMREVFodFo6NKli3m5u7s7LVq0MP/u+vTpw6xZs7h+/Tpbt24lPDwcb29vIiMjmTJlCrt27TJHXx05coTMzEzc3d0txpCTk2ORNunv74+np2eF9yEmJoZXXnmFPXv2kJiYaI4ou3jxIiEhIURFRdGmTRuLv5cbj2vbtm0JDg5m1apVPP/882zdupWEhARGjx5tbvPvv//yzjvvcOrUKdLT09Hr9eTm5pKdnW2OnLOzszOfe2D5t5CQkEBcXBz9+vUrcT8qcnyioqL4v//7P4vl3bp1KxYZamurXEtM6a+3ivoZWQY4SspNut6QUsMjEVQVk7m/o9GIUZawtr215v4mrG2cmdhAuQB9fWkjel35aZP/HfycK2pwNMr06/JUiW3cCwRcR811URGzlmGKLLMzzf76dSEhvXr9yqqCc69nGZ2jRA/pJGii03Nfv9JnGrsXCLknsi7jYqdcI40yZAgfvTqFKQ3TO88klimRZSfjlAe0pp4O2Ggrd3PduYkbUr4bALGZl81RZb2aezCtlxJl+dafUfxa8HlZKZggIsvqC9XlWeZj74OjVnmQHdxkMIvvWgxATGrZYllsWiwjfhvBU5Elfy8L6j6mAj13UhpmYSXMIub
```

Comment:
Change this to one `apply_availability`, `apply_constraint`, or `apply_tracker_inactive` as operational issues that are turned on and off as needed.

Terrain shading isn't a great example as it isn't the type of thing you fit to your site. Terrain shading in general improves accuracy but isn't on by default to maintain existing behaviour, so if it was a new site or integration we'd suggest turning it on but haven't forced that on existing users.

Reply:
If we replace the live examples and need to rerun them now, it is proving hard to find any suitable snow examples, given the lateness in the season and the anomalous heat.  I think best options are either

1. keep the snow soiling example we have and just axe the terrain shading one
2. ditch both examples and do two new ones - say apply_availability and apply_tracker_inactive

Though I think the snow example is a nice one, it has the downside that if people are trying the example notebook for their own site, they most likely won't have any snow to see the effect of.

Thoughts?

Reply:
I like having two examples but also agree the snow is nice.

Axe terrain shading and add some text mentioning that there are other apply... parameters for things like controlling availability

Reply:
ok